src/channel/handlers.js

```diff
@@ -28,6 +28,12 @@ import {
   fsmId
 } from './internal'
 import { unpackTx, buildTx } from '../tx/builder'
+import {
+  IlleagalArgumentError,
```

Comment:
```suggestion
  IllegalArgumentError,
```

we already had this error type in the SDK, right? we should fix the spelling if we do a major release anyway 😅

Reply:
@marc0olo We had no such error in the SDK. No no, It's unacceptable irrespective of release. I will fix the spelling.

Reply:
Fixed it.